src/System.CodeDom/src/Microsoft/VisualBasic/VBCodeGenerator.cs

```diff
@@ -0,0 +1,2931 @@
+                "object",
+                "option",
+                "orelse",
+                "public",
+                "resume",
+                "return",
+                "select",
+                "shared",
+                "single",
+                "static",
+                "string",
+                "typeof",
+                "ushort",
+            },
+            new string[] { // 7 characters
+                "andalso",
+                "boolean",
+                "cushort",
+                "decimal",
+                "declare",
+                "default",
+                "finally",
+                "gettype",
+                "handles",
+                "imports",
+                "integer",
+                "myclass",
+                "nothing",
+                "partial",
+                "private",
+                "shadows",
+                "trycast",
+                "unicode",
+                "variant",
+            },
+            new string[] {  // 8 characters
+                "assembly",
+                "continue",
+                "delegate",
+                "function",
+                "inherits",
+                "operator",
+                "optional",
+                "preserve",
+                "property",
+                "readonly",
+                "synclock",
+                "uinteger",
+                "widening"
+            },
+            new string [] { // 9 characters
+                "addressof",
+                "interface",
+                "namespace",
+                "narrowing",
+                "overloads",
+                "overrides",
+                "protected",
+                "structure",
+                "writeonly",
+            },
+            new string [] { // 10 characters
+                "addhandler",
+                "directcast",
+                "implements",
+                "paramarray",
+                "raiseevent",
+                "withevents",
+            },
+            new string[] {  // 11 characters
+                "mustinherit",
+                "overridable",
+            },
+            new string[] { // 12 characters
+                "mustoverride",
+            },
+            new string [] { // 13 characters
+                "removehandler",
+            },
+            // class_finalize and class_initialize are not keywords anymore,
+            // but it will be nice to escape them to avoid warning
+            new string [] { // 14 characters
+                "class_finalize",
+                "notinheritable",
+                "notoverridable",
+            },
+            null,           // 15 characters
+            new string [] {
+                "class_initialize",
+            }
+        };
+
+        internal VBCodeGenerator()
+        {
+        }
+
+        internal VBCodeGenerator(IDictionary<string, string> providerOptions)
+        {
+            _provOptions = providerOptions;
+        }
+
+        /// <devdoc>
```

Comment:
Nit: Could these `<devdoc>` be converted to simple `<summary>`? Applies throughout.